functions/process.py

```diff
@@ -0,0 +1,70 @@
+import pandas as pd
+
+def replace_ensembl_ids(expression_df,
+gene_id_mapping):
+
+    """
+    Replaces ensembl gene ids with hgnc symbols 
+    
+    Arguments
+    ---------
+    expression_df: df
+        gene expression data matrix (sample x gene)
+    gene_id_mapping: df
+        Dataframe mapping ensembl ids (used in DE_stats_file) to hgnc symbols,
+        used in Crow et. al.
+    """
+    # Read in data
+    gene_expression = expression_df
+
+    # Different cases of many-many mapping between gene ids
+    #count = 0
+    #symbols = []
+    #for symbol, group in gene_id_mapping.groupby("ensembl_gene_id"):
+    #    if group['hgnc_symbol'].nunique() > 1:
+    #        print(group)
+    #        count += 1
+    #        symbols.append(symbol)
+    #count
```

Comment:
Is this commented out code no longer necessary?

Reply:
This code isn't being used no, it was more of a note to myself of how I discovered the different cases that needed to be addressed when mapping gene ids